src/main/java/com/it4045/pokemonviewer/enterprise/dto/Pokedex.java

```diff
@@ -6,5 +6,5 @@
 public @Data
 class Pokedex {
 
-    private List<Pokemon> pokemon;
+    private List<Pokemon> allPokemon;
```

Comment:
In this portion I changed pokemon to allPokemon because it's singular and plural and could be confusing. There are also other variables that are in other classes named pokemon.

Reply:
That makes sense.  pokemon is singular, while a list is plural.  Some kind of plural indication is a good idea.